.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: ruby
+rvm:
+  - 2.2.4
+# bundler_args: --without production
+# script:
+#   - bundle exec rake db:test:prepare
+#   - bundle exec rspec spec/
```

Comment:
I think travis will fail like this (it will use its default and will run all the specs, i imagine.  It may be that there is no benefit to running travis since this is just a file of specs ...  what do you think of taking travis OUT for now?